docs/transfer.md

```diff
@@ -0,0 +1,68 @@
+# Transferring data to the IKIM cluster
+
+`Nota bene`: The cluster can accomodate only de-identified data,  no directly patient related data can be uploaded. All PII personal identifying information has to be removed prior to upload.
+
+The cluster provices a number of storage facilities described
+[here](./storage.md).
+
+## Introduction to data transfer
+
+Data transfer across the network offers a great way to avoid the use of sneakernet (i.e. carrying hard drivers around campus to move data from A to B). We strongly advise reading the info on the cluster storage prior to moving forward here.
+
+Larger scale data transfer requires some degree of familiarity with the technologies available and the sending and receiving systems.
+
+We provide three different means for data transfer. We note that for larger transfers, the speed of the device the data is stored on remotely makes a difference.
+
+### Using a web browser to move data into the cluster
```

Comment:
I don't think we should mention things that aren't ready.

Reply:
I am debating this as well, lets comment it out...

Reply:
I am debating this as well, lets comment it out...